skiko/src/jvmMain/kotlin/org/jetbrains/skiko/Convertors.kt

```diff
@@ -76,3 +77,59 @@ fun BufferedImage.toBitmap(): Bitmap {
 fun BufferedImage.toImage(): Image {
     return Image.makeFromBitmap(toBitmap())
 }
+
+fun toSkikoEvent(e: MouseEvent): SkikoPointerEvent {
+    return SkikoPointerEvent(
+        e.x.toDouble(),
+        e.y.toDouble(),
+        when(e.button) {
+            MouseEvent.BUTTON1 -> SkikoMouseButtons.LEFT
+            MouseEvent.BUTTON2 -> SkikoMouseButtons.RIGHT
+            MouseEvent.BUTTON3 -> SkikoMouseButtons.MIDDLE
+            else -> SkikoMouseButtons.NONE
+        },
+        when(e.id) {
+            MouseEvent.MOUSE_PRESSED -> SkikoPointerEventKind.DOWN
+            MouseEvent.MOUSE_RELEASED -> SkikoPointerEventKind.UP
+            MouseEvent.MOUSE_DRAGGED -> SkikoPointerEventKind.DRAG
+            MouseEvent.MOUSE_MOVED -> SkikoPointerEventKind.MOVE
+            MouseEvent.MOUSE_ENTERED -> SkikoPointerEventKind.ENTER
+            MouseEvent.MOUSE_EXITED -> SkikoPointerEventKind.EXIT
+            else -> SkikoPointerEventKind.UNKNOWN
+        },
+        e
+    )
+}
+
+fun toSkikoEvent(e: MouseWheelEvent): SkikoPointerEvent {
+    return SkikoPointerEvent(
+        e.x.toDouble(),
+        e.y.toDouble(),
+        SkikoMouseButtons.NONE,
```

Comment:
We can know which button is pressed via `e.modifiersEx and InputEvent.BUTTON1_DOWN_MASK != 0`

Reply:
+1